app/queries/servers/post.ts

```diff
@@ -279,3 +279,42 @@ export const countUsersFromMentions = async (database: Database, mentions: strin
     const [groups, usersCount] = await Promise.all([groupsQuery, usersQuery]);
     return groups.reduce((acc, v) => acc + v.memberCount, usersCount);
 };
+
+export const queryPostsWithPermalinkReferences = async (
+    database: Database,
+    referencedPostId: string,
+    channelId?: string,
```

Comment:
why is this optional? or needed at all ?

Reply:
This is not required. Removed it.